libnavui-dropin/src/main/res/layout/mapbox_navigation_view_layout.xml

```diff
@@ -20,13 +20,13 @@
     -->
     <androidx.coordinatorlayout.widget.CoordinatorLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:fitsSystemWindows="true">
+        android:layout_height="match_parent">
 
         <androidx.coordinatorlayout.widget.CoordinatorLayout
             android:id="@+id/coordinatorLayout"
             android:layout_width="match_parent"
-            android:layout_height="match_parent">
+            android:layout_height="match_parent"
```

Comment:
Do we need the inner `CoordinatorLayout` since we are moving `FrameLayout` containing `InfoPanel` to the outer `CoordinatorLayout`?

Reply:
Unfortunately, yes. `com.google.android.material.bottomsheet.BottomSheetBehavior` can be only attached to a view that's a direct child of the `CoordinatorLayout`.

Reply:
Can't we simply register a window insets listener and offset the content as necessary?

Reply:
That's exactly what [`android:fitsSystemWindows="true"`](https://developer.android.com/reference/android/view/View#attr_android:fitsSystemWindows) does.

Reply:
I know, but using `CoordinatorLayout` just for that purpose looks like overkill to me. 
Also I remember problems with `fitsSystemWindows` option, when it consumed the insets and didn't allow any other views to receive them. 
I guess we can leave it your way and improve later if we encounter a problem with this approach.

Reply:
Agreed. Let's keep an eye on this.